server/api/index.js

```diff
@@ -1,8 +1,14 @@
 const Shopify = require('shopify-api-node');
 const express = require('express');
 const router = express.Router();
```

Comment:
Now that you've moved the route out of this file, `router` is unused. Also, does the file name make sense now?